Gemfile

```diff
@@ -74,6 +74,7 @@ group :development, :test do
   gem 'faker', '~> 2.6.0'
   gem 'pry-rails', '~> 0.3.9'
   gem 'action-cable-testing'
+  gem 'brakeman'
```

Comment:
Looks great @StlMaris123 
But I think brakeman is used only to detect vulnerabilities right? 
So it should be better to add this to the development group if it isn't being used in production.